src/js/components/form-elements/LabelField.jsx

```diff
@@ -23,7 +23,7 @@ const LabelField = (props) => {
               <label htmlFor={attr.id} className="col-md-2 col-form-label col-form-label-xs text-right">{ FieldLabel }</label> :
               <FieldLabel />
           }
-        <div className="col-md-4 align-self-center">
+        <div className="col-md-4 align-self-center font-size-xs">
```

Comment:
this is used when LabelField is rendered outside the table, I'm not sure if this should be changed

Reply:
You're right, it shouldn't. I forgot to remove this change.